docs/API.md

```diff
@@ -1551,6 +1555,44 @@ Example: `GET ~bob/trusty/wordpress-42/meta/id-series`
 }
 ```
 
+#### GET *id*/meta/home-page
+
+The `meta/home-page` path returns the home page project of 
+the entity if any specified.
+
+```go
+type HomePageResponse struct {
+        HomePage string
+}
+```
+
+Example: `GET bundle/mediawiki/meta/home-page`
```

Comment:
@urosj  Do we want a DELETE ? 
I have HomePage and BugsURL as url.URL element so it will be an empty element. So a PUT of an empty element will be a DELETE